enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,305 @@
+- [ ] End user documentation
+- [ ] Tile up
+- [ ] CI e2e jobs created to cover installation against this platform
+- [ ] Plan defined & support engaged to gather feedback from users
+- [ ] Sufficient time for feedback (6 weeks from start of Tech Preview)
+
+#### Tech Preview -> GA
+- [ ] CI has been upgraded to cover testing of modifications to the installer (PR validation)
+  - This will likely require PROW running on a Power build farm, or upgrades to PROW to allow for non-x86 builds
+  - CI "soak" for 6 weeks to ensure a stable baseline
+- [ ] Final optimizations for install time
+- [ ] Expanded testing (upgrade, downgrade, scale, load/performance)
+- [ ] Documentation is finalized
+
+#### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+All upgrade and downgrade expectations are the same for existing IPI deployments.
+
+### Version Skew Strategy
+
+This should be the same as other IPI deployments.
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation History`.
+
+## Drawbacks
+
+The greatest argument against IPI on Power VS would be opportunity cost. That said, for the Multi-Arch team, we believe
+this to be the best return on investment when compared to the many other platforms asked of our team.
+
+## Alternatives
+
+The are other options for supporting IPI for for power hardware. One option we use internally is IPI on KVM via libvirt;
+however, as this is not supported outside of development purposes for x86 and KVM adoption on power goes against the
+market preference for PowerVM based virtualization, we believe that targeting the IBM Cloud is the best option we have
+available to us currently. PowerVM and PowerVC are also options; however, both require expanding RHEL support to new
+hypervisors, which poses long term sustainability challenges.
+
+## Infrastructure Needed
+
+Many on the Multi-Arch team have already gotten development access to the IBM Cloud. Involved parties should coordinate
+access with rsene@redhat.com.
+
+[ibm-cloud-website]: https://www.ibm.com/cloud?p1=Search&p4=43700051838804081&p5=e&gclid=EAIaIQobChMIn77Qwqqi8AIV2_6zCh3gdAe8EAAYASAAEgJV9_D_BwE&gclsrc=aw.ds
+[power-vs-website]: https://www.ibm.com/cloud/power-virtual-server
+[direct-link-website]: https://www.ibm.com/cloud/direct-link
+[power-vs-reference-doc]: https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server
+[vpc-loadbalancer-website]: https://cloud.ibm.com/docs/vpc?topic=vpc-nlb-vs-elb
+[ibm-cloud-terraform-provider]: https://github.com/IBM-Cloud/terraform-provider-ibm
+[openshift-machine-operator]: https://github.com/openshift/machine-api-operator
+[power-go-client]: https://github.com/IBM-Cloud/power-go-client
+[ibm-cloud-dns-website]: https://www.ibm.com/cloud/dns
```

Comment:
this has changed, so the link should now point to https://cloud.ibm.com/catalog/services/internet-services#about or https://cloud.ibm.com/docs/cis?topic=cis-about-ibm-cloud-internet-services-cis (or both?)